### ISSUE TITLE
[Bug] ComparisonMessage.DocUrl are broken link

### ISSUE THREAD

**Issue by vernou** [criteo/openapi-comparator#6]
The doc url displayed from a comparison message is broken. Example :
https://github.com/Azure/openapi-diff/tree/master/docs/rules/10021.md
https://github.com/Azure/openapi-diff/tree/master/docs/rules/10281.md

To reproduce :
```csharp
...
var comparator = new OpenApiComparator();
var messages = comparator.Compare(oldFileName, oldOpenApiSpec, newFileName, newOpenApiSpec);
foreach(var message in messages)
{
    Console.WriteLine(message);
}
```
Output :
```raw
..., docurl = https://github.com/Azure/openapi-diff/tree/master/docs/rules/10021.md, mode = Removal
..., docurl = https://github.com/Azure/openapi-diff/tree/master/docs/rules/10281.md, mode = Update
..., docurl = https://github.com/Azure/openapi-diff/tree/master/docs/rules/10281.md, mode = Update
```

It seems that the problem comes from [ComparisonMessage._docBaseUrl](https://github.com/criteo/openapi-comparator/blob/0.4.0/src/Criteo.OpenApi.Comparator/ComparisonMessage.cs#L19). The base doc url is "https://github.com/Azure/openapi-diff/tree/master/docs/rules".

All rules are replicated in this repository, I suggest that comparison message doc url redirect directly to "https://github.com/criteo/openapi-comparator/tree/{version}/documentation/rules".

Do you want some help? I can do the pull request.

**Comment by PaulMathon** [comment]
Hi @vernou, thanks for reporting this issue, indeed the issue seems to come from the original fork of the [openapi-diff](https://github.com/Azure/openapi-diff) repository, your proposal should fix it. I let you create a pull request. 
Thank you!

**Comment by PaulMathon** [comment]
The fix has been released in version [0.5.1](https://github.com/criteo/openapi-comparator/releases/tag/0.5.0). 
Please note that other changes have been introduced in the release, you may have to update the way you're using the comparator.
